babyai/bot.py

```diff
@@ -317,154 +317,6 @@ def take_action(self, action):
         return True
 
 
-class GoToObjSubgoal(Subgoal):
-    def __init__(self, bot=None, datum=None, adjacent=False):
-        '''If adjacent=True, then the bot would only go two cells away from the objective (with free cell in between)'''
```

Comment:
I think this comment should be rewritten/clarified. If adjacent is set, then the bot *will* go two cells away? What do you mean by "with free cell in between". It will leave a free cell in between it and the objective?